app/controllers/FaciaToolController.scala

```diff
@@ -246,11 +247,13 @@ class FaciaToolController(
 
   def pressLivePath(path: String) = AccessAPIAuthAction { request =>
     faciaPressQueue.enqueue(PressJob(FrontPath(path), Live, forceConfigUpdate = Option(true)))
+    FaciaPressTopic.publish(PressJob(FrontPath(path), Live, forceConfigUpdate = Option(true)))
```

Comment:
Using the logs, we couldn't see that this function was being invoked in the workflow that we were testing, but because, at some point, the plan is to remove the SQS code with SNS, we decided to add the call to the SNS topic publish function here as well.

Reply:
Briefly tried following the trail, I'm not 100% sure either, but it looks like there's a view for fronts which are considered "stale"; this includes a button that enables repressing the front, which would then trigger this logic, _I think_. Not sure if this actually gets used, but seems worth adding here too!